client/src/components/scans/scanJobsList.js

```diff
@@ -61,9 +61,8 @@ class ScanJobsList extends React.Component {
   }
 
   render() {
```

Comment:
Note:  Another, we could embrace a little bit more on the stateless component side with this one, next round